src/jshal.js

```diff
@@ -56,7 +56,7 @@ mergeInto(LibraryManager.library, {
   },
 
   mp_js_hal_filesystem_name: function (idx, buf) {
-    const name = fs.name(idx);
```

Comment:
This was recently broken by #51.
This PR seems to be working well so I'm going to go go ahead and merge shortly.